src/services/search.service.ts

```diff
@@ -164,6 +164,19 @@ export class SearchService implements SearchServiceAbstraction {
             if (c.login && c.login.uri != null && c.login.uri.toLowerCase().indexOf(query) > -1) {
                 return true;
             }
+
+            if(c.hasFields) {
```

Comment:
Please check your VS Code + VS macOS formatting settings to ensure for JS/TS there's a space between control keywords and the open parens on these blocks.

Reply:
Sorry about that. I'm a Vim guy, still working out some discrepancies between my formatting setup & the VS Code defaults. Should be resolved now.